authui/src/web3.ts

```diff
@@ -192,6 +192,7 @@ export class WalletConfirmationController extends Controller {
   }
 
   connect() {
+    this.displayedTarget.textContent = "-";
```

Comment:
Better to put the dash in the template.

And we should be consistent in where to set this value. For example, when I disconnect, it does not show a dash again. Ideally, the textContent should be updated in the same function. This would eliminate a lot of potential inconsistent UI state changes.

```typescript
function _getAccount() {
  this.displayedTarget.textContent = "-";
  // ... remaining code
  // textContent must either be "-" or some address after this function returns.
}
```